tools/depends/target/Makefile

```diff
@@ -28,6 +28,9 @@ endif
 
 ifeq ($(OS),ios)
   EXCLUDED_DEPENDS = libcec libusb
+ifeq ($(CPU),arm64)
+  DEPENDS += boblight
```

Comment:
we build libboblight and bundle it - it needs to get signed during installation on tvos - i can add an check for tvos somehow i guess to make this not that missleading